packages/frontend/src/components/PopupWrapper.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Popup } from 'react-leaflet';
+
+/* eslint-disable react/prop-types */
+export const PopupWrapper = ({ id, details }) => {
+    return (
+        <Popup>
+            <div>
+                <h2>{id}</h2>
```

Comment:
would be nice if we add icons aswell. or add some css with pure css or using the MUI library